terasoluna-gfw-functionaltest-selenium/src/test/java/org/terasoluna/gfw/functionaltest/app/transactiontoken/TransactionTokenTest.java

```diff
@@ -69,8 +59,6 @@ public void setUp() {
             quitDefaultWebDriver();
         }
         bootDefaultWebDriver();
-        driver = new EventFiringWebDriver(getDefaultWebDriver());
-        driver.register(new ScreenCaptureWebDriverEventListener(screenCapture));
```

Comment:
`EventFiringWebDriver` & `WebDriverEventListener` has bean deprecated, but should you remove the screen capture feature?

Reply:
No captures were taken because `selenium.enableCapture=false`. There is no visual confirmation of the capture in the test item either. Also, since the captcha is obtained in a separate class when the test fails, we believe it is safe to remove the `ScreenCaptureWebDriverEventListener`.

Reply:
`ScreenCaptureWebDriverEventListener` captures after click and after page transition, not on test failure. There is no perfect replacement by another mechanism.
Whether or not this mechanism is necessary should be discussed in a new Issue. At least I don't think the presented reason are good enough to remove the mechanism itself.（I'm not strongly against removing it.）

Reply:
Created #985